### PR TITLE
Adding support for running instrumentation and appium tests in custom Mode, Also added support for running appium Python Tests from the plugin

### PR DESCRIPTION
**Pull request by anurag1408**
… mode, Also added support for apium python

### CODE REVIEW

File: aws-devicefarm-gradle-plugin/build.gradle [hunk 1]
```diff
@@ -49,7 +49,7 @@ test {
 }
 
 group 'com.amazonaws'
-version '1.4-SNAPSHOT'
```
Comment (nikhil-dabhade) [inline aws-devicefarm-gradle-plugin/build.gradle]: I think we should keep this as is. The release process updates this.

File: aws-devicefarm-gradle-plugin/src/main/groovy/com.amazonaws.devicefarm/extension/CustomMode.groovy [hunk 2]
```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (nikhil-dabhade) [inline aws-devicefarm-gradle-plugin/src/main/groovy/com.amazonaws.devicefarm/extension/CustomMode.groovy]: Needs to change to 2015-2018

**Comment by fernandolucheti** [comment]
Is there any prediction when this is going to be merged/available?
I do need to use custom environment on my instrumentation tests, it would be quite useful.

**Comment by mg6maciej** [comment]
Is it not released yet? From the code change it looks like it would go to 1.4, but 1.3 is the latest version released despite [the same readme mentioning possibility to use `testSpecName`](https://github.com/awslabs/aws-device-farm-gradle-plugin/blob/master/README.md#instrumentation).

**Pull request merged**